qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -193,11 +294,14 @@ def clifford_1_qubit_circuit(cls, num):
         if p == 3:
             qc.z(0)
 
+        if basis_gates:
+            qc = _transform_clifford_circuit(qc, basis_gates)
+
```

Comment:
I think there is no point in transforming to basis gates here. This method `clifford_1_qubit_circuit` is meant to return a Clifford as is and not transpiled. This is part of the "old" code. The equivalent method that returns the transpiled clifford is `transpiled_clifford_from_num_1q`. 
You can see for example that the method `clifford_1_qubit` : 

```
def clifford_1_qubit(cls, num):
          return Clifford(cls.clifford_1_qubit_circuit(num), validate=False)
```
So it will simply create a Clifford again out of the transpiled circuit you created. 
The main place where this method is used is in `create_clifford_mapping.py`, and there the Cliffords do not need to be transpiled.

In addition, this method should probably be united with `transpiled_clifford_from_num_1q`, but I preferred not to do this in the current PR. This is something of a sanity check to see that both methods generated the same Cliffords.
All of this is true also for `clifford_2_qubit_circuit`.

Reply:
I'm going to refactor APIs in `clifford_utils` in a follow-up PR (you can see [the draft](https://github.com/itoko/qiskit-experiments/pull/2)). I'd like to discuss these points on the PR.